ci/packet/packet-cluster.lokocfg.envsubst

```diff
@@ -226,3 +226,26 @@ component "web-ui" {
 }
 
 component "inspektor-gadget" {}
+
+component "velero" {
+  provider = "restic"
+
+  restic {
+    credentials = <<EOF
+[default]
+aws_access_key_id = "$AWS_ACCESS_KEY_ID"
+aws_secret_access_key = "$AWS_SECRET_ACCESS_KEY"
+EOF
+
+    backup_storage_location {
+      provider = "aws"
+      bucket   = "velero-test-ci"
```

Comment:
This bucket must exist for tests to converge? If so, I think this dependency is quite hidden and we should make it somehow more explicit or document the requirement.

Reply:
Added a comment on top of this parameter.